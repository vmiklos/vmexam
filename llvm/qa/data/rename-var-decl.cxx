class C
{
public:
    static const int aS[];
};
