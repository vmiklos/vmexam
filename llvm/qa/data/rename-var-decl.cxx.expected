class C
{
public:
    static const int m_aS[];
};
